server/modules/surveyRdb/service/surveyRdbService.js

```diff
@@ -1,125 +1,28 @@
-const R = require('ramda')
-const fastcsv = require('fast-csv')
-
 const Survey = require('../../../../core/survey/survey')
-const NodeDef = require('../../../../core/survey/nodeDef')
-const NodeDefTable = require('../../../../common/surveyRdb/nodeDefTable')
-
-const DataTable = require('../schemaRdb/dataTable')
 
 const SurveyManager = require('../../survey/manager/surveyManager')
 const SurveyRdbManager = require('../manager/surveyRdbManager')
-const RecordManager = require('../../record/manager/recordManager')
 
-const _getQueryData = async (surveyId, cycle, nodeDefUuidTable, nodeDefUuidCols = []) => {
-  const surveySummary = await SurveyManager.fetchSurveyById(surveyId, true)
-  const surveyInfo = Survey.getSurveyInfo(surveySummary)
+const _getSurvey = async (surveyId, cycle) => {
```

Comment:
it's more a "fetch" than a get...

Reply:
I can rename it after we merge this branch into processing chain poc